NEW_PROJECTS.md

```diff
@@ -0,0 +1,70 @@
+# Creating a new SPIFFE Project
+
+We are grateful for your desire to expand the SPIFFE ecosystem with your contribution!
+
+This is an evolving set of guidelines and instructions for how to bring your existing project, or idea for a completely fresh project, under the SPIFFE organization.
+
+If your project is already under the SPIFFE banner, you may be looking for [MATURITY](/MATURITY.md) instead.
+
+## Notice
+
+Maintainer groups are given significant freedom in managing their projects to that we may best foster growth and innovation. There are some elements of SPIFFE subproject ownership to be aware of:
+The [SPIFFE Steering Committee](/ssc/README.md) (SSC) governs the SPIFFE organization as a whole, and has the final authority on acceptance, rejection, or eviction of projects. Additionally, as stewards of community health, the SSC is responsible for intervening in any [Code of Conduct](/CODE-OF-CONDUCT.md) violations or derelictions of maintainer responsibility.
+Note that when contributing a project to SPIFFE, its purpose becomes to serve the SPIFFE community rather than any personal gain. Additionally, we expect a sufficient number of maintainers for a project as it grows, not all of whom may bring the same direction you may originally have had in mind.
+Carefully consider this change in control and purpose.
+
+*Every SPIFFE project* at minimum must follow the [SPIFFE Code of Conduct](/CODE-OF-CONDUCT.md).
+
+## Project Proposal
+
+### Contact
+
+The SSC can be contacted with proposal details at ssc@spiffe.io or on [our SPIFFE slack channel](https://spiffe.slack.com/archives/C01GQ267JJU).
+
+### Proposal Outline
+
+The following details are expected for proposals to help us best understand how the project will serve the community.
+
+Additionally, the information here make great initial `README.md`, `CONTRIBUTING.md`, and `GOVERNANCE.md` files for the repo!
+
+#### Initial Maintainer List
+
+The list of initial maintainers of the project, including GitHub handle.
+
+#### Project Purpose and Direction
```

Comment:
we can consider providing example questions that can guide contributors in framing their project’s objectives and vision.

Reply:
suggestions to add beyond what's here?
I also want to provide as much guide here as possible; current list is all I could initially think of but happy to add more

Reply:
For purpose

- What specific pain point does your project address?
- What are the current workarounds or alternative solutions, and why are they inadequate?
- Who are the primary end-users or beneficiaries of this project?
- How does your project align with SPIFFE's overall mission and goals?

For Direction
- What is the project’s roadmap for the next 6-12 months?
- How will the project scale? What are the scalability goals?
- Are there any legal or ethical considerations?
- What are potential challenges, and how do you plan to overcome them?
- What kind of community involvement do you foresee (e.g., are there opportunities for novice contributors, or is the project more suitable for experts in the field?)
- What technologies will the project leverage, and are they aligned with the larger ecosystem?
- How will the project maintain data integrity, security, and privacy?
- Are there any possible collaborations with existing SPIFFE projects?
- How will your project deal with failure scenarios or fallbacks?
- 
Providing these questions as a guide can help contributors delve deeper into their project's objectives and execution plan, offering a more comprehensive view for the review committee.

But on the flipside; adding too much of these questions can feel like a burden.
I’m not sure, maybe just a subset of the most important ones?

Reply:
added